src/main/java/uk/gov/pay/api/model/telephone/CreateTelephonePaymentRequest.java

```diff
@@ -0,0 +1,162 @@
+package uk.gov.pay.api.model.telephone;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.pay.api.validation.ValidCardExpiryDate;
+import uk.gov.pay.api.validation.ValidCardFirstSixDigits;
+import uk.gov.pay.api.validation.ValidCardLastFourDigits;
+import uk.gov.pay.api.validation.ValidCardType;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class CreateTelephonePaymentRequest {
+
+    public static final int REFERENCE_MAX_LENGTH = 255;
+    public static final int AMOUNT_MAX_VALUE = 10000000;
```

Comment:
I'm a bit uneasy about how we're just scattering this round our codebase as a hardcoded value, although I realise you are following precedent here. Just consolidating this into a single static class would be a good start, maybe in a follow up PR, or alternatively create an Amount type that can be validated and use it everywhere

Reply:
I agree - I'll do this in a follow up PR.